nixos/modules/services/databases/aerospike.nix

```diff
@@ -42,7 +42,11 @@ in
       enable = mkEnableOption "Aerospike server";
 
       package = mkOption {
-        default = pkgs.aerospike;
+        default =
```

Comment:
Wouldn't specifying `defaultText` be a better solution? Then no one would need to build `aerospike` to have access to the NixOS manual.

Reply:
Right! I knew there was a way, but completely forgot what it was. Thanks!